cmake/procstat-config.cmake

```diff
@@ -0,0 +1,27 @@
+include(FindPackageHandleStandardArgs)
+include("$ENV{WORKSPACE_TOP}/common/cmake/lb.cmake")
+
+lb_local_path(PROCSTAT_BUILD procstat procstat)
+
+find_path(Procstat_INCLUDE_DIR
+  "procstat.h"
+  PATHS "$ENV{WORKSPACE_TOP}/procstat"
+  PATH_SUFFIXES "src"
+)
+
+find_library(Procstat_LIBRARY NAMES procstat PATHS "${PROCSTAT_BUILD}/lib")
+
+lb_version(Procstat_VERSION procstat procstat)
```

Comment:
are you using `Procstat_VERSION` or is it for the caller?

Reply:
It looks like the best practice in CMake.

You can't tell CMake because he disapproves non-numeric version number